src/index.less

```diff
@@ -33,9 +33,15 @@ body,
 
 .error-boundary {
   min-width: 400px;
+  min-height: 125px;
```

Comment:
Is this actually needed? This would probably result in a larger box than actually required if the `RerouteToLogin` component isn't passed to the error box (e.g. if not an auth error).

Reply:
Not needed, I have deleted it.